pkg/lint/lintersdb/manager.go

```diff
@@ -992,6 +956,42 @@ func (m Manager) GetAllLinterConfigsForPreset(p string) []*linter.Config {
 	return ret
 }
 
+func linterConfigsToMap(lcs []*linter.Config) map[string]*linter.Config {
+	ret := map[string]*linter.Config{}
+	for _, lc := range lcs {
+		lc := lc // local copy
```

Comment:
this is copy of pointer – no make sense?

Reply:
it was existing code, fixed in the next PR.